frontend/src/entities/jobs/training/create/training-job-create.tsx

```diff
@@ -418,7 +418,16 @@ export default function TrainingJobCreate () {
                             disabled={state.formSubmitting}
                             variant='primary'
                             onClick={() => {
-                                handleSubmit();
+                                const parseResult = formSchema.safeParse(state.form);
+                                if (parseResult.success) {
+                                    handleSubmit();
+                                } else {
+                                    setState({
+                                        type: 'updateState',
+                                        payload: { validateAll: true },
+                                    });
+                                    scrollToInvalid();
+                                }
```

Comment:
We usually do this in the handleSubmit() call. Is there a reason you opted to inline it here?

Reply:
no particulae reason. this was copy paste from hpo jobs: https://github.com/awslabs/mlspace/blob/143dc6003b2c1193c589be1aef3f5820a73d9218/frontend/src/entities/jobs/hpo/create/hpo-job-create.tsx#L451